bin/dune

```diff
@@ -39,6 +43,8 @@
 
 (install
  (section bin)
+ (enabled_if
```

Comment:
Could we add a comment explaining why these `enabled_if` are necessary?

Reply:
done